eoscompanion/dummy_feed.py

```diff
@@ -143,16 +140,14 @@ def dummy_feed():
                         },
                         {
                             'contentType': 'quoteOfTheDay',
-                            'source': [
-                                {
-                                    'type': 'application',
-                                    'detail': [
-                                        {
-                                            'applicationId': 'com.endlessm.quote_of_the_day.en'
-                                        }
-                                    ]
-                                }
-                            ],
+                            'source': {
+                                'type': 'application',
+                                'detail': [
```

Comment:
detail should also be one object, not a list of objects.

Reply:
That's the layout in the content flow document. I presume to support different sources than applications, which might have other details.

<code>
“sources”: [
            {
                “source”: {
                    “type”: [one of “application”],
                    “detail”: [
                        case type == “application”: {
                            “applicationId”: [machine-readable application ID]
                        }
                    ]
                },
</code>